helloapache/README.md

```diff
@@ -1,73 +1,57 @@
+```sh
+sudo atomicapp run projectatomic/helloapache
 ```
-[sudo] atomic run projectatomic/helloapache
-```
-
-Note: This option is not interactive because all params in the Nulecule file have default values.
-
-## Option 2: Unattended
-
-1. Create the file `answers.conf` with these contents:
-
-    This sets up the values for the two configurable parameters (image and hostport) and indicates that kubernetes should be the orchestration provider.
-
-        [general]
-        provider = kubernetes
-
-        [helloapache-app]
-        image = centos/httpd # optional: choose a different image
-        hostport = 80        # optional: choose a different port to expose
-1. Run the application from the current working directory
-
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
-
-
-1. As an additional experiment, remove the kubernetes pod and change the provider to 'docker' and re-run the application to see it get deployed on base docker.
 
-## Option 3: Install and Run
-
-You may want to download the application, review the configuraton and parameters as specified in the Nulecule file, and edit the answerfile before running the application.
-
-1. Download the application files using `atomic run IMAGE --mode fetch`
-
-        [sudo] atomic run projectatomic/helloapache --mode fetch
+`atomic`
+```sh
+sudo atomic run projectatomic/helloapache
+```
 
-1. Rename `answers.conf.sample`
+__Unattended deployment__
 
-        mv answers.conf.sample answers.conf
+`atomicapp`
+```sh
+atomicapp fetch projectatomic/helloapache --destination helloapache
+cd helloapache
+atomicapp genanswers .
+atomicapp run -a answers.conf .
+```
 
-1. Edit `answers.conf`, review files if desired and then run
+`atomic`
+```sh
+sudo atomic run projectatomic/helloapache --mode fetch --destination helloapache
+cd helloapache
+sudo atomic run projectatomic/helloapache --mode genanswers answers .
+sudo atomic run projectatomic/helloapache -a answers.conf .
+```
 
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
+### Interaction
 
-## Test
-Any of these approaches should create a kubernetes pod or a running docker container. 
+__Docker provider (ex. --provider=docker):__
 
-With a kubernetes pod, once its state is "Running" curl the minion it's running on.
+The default port is :80.
 
-```
-$ kubectl get pod helloapache
-POD                IP                  CONTAINER(S)       IMAGE(S)           HOST                LABELS              STATUS
-helloapache        172.17.0.8          helloapache        centos/httpd       10.3.9.216/         name=helloapache   Running
-$ curl 10.3.9.216
-<bunches_of_html_goodness>
+```sh
+curl localhost
+<html-foobar>
 ```
 
-If you test the docker provider, once the container is running, curl the port on your localhost.
+__Kubernetes provider (default):__
 
-```
-$ curl localhost
-<bunches_of_html_goodness>
+By default kubernetes will assign an available external IP.
```

Comment:
Hence why I add `kubectl get pod helloapache` :).

Yeah, it's enough since that's what the previous README specified / the only way to get the external-ip info.